shrine/lib/main.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018-present the Flutter authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:flutter/material.dart';
+import 'package:scoped_model/scoped_model.dart';
+
+import 'app.dart';
```

Comment:
I think we found you need to `package:` this import because of a sometimes bug in Dart.

Reply:
Only in main tho.

Reply:
I'll double-check. The only package-or-not issue I've run into so far, though, is that if you import a file twice in your app, once with `package` and once without, Dart will consider them two separate namespaces and double up all the exports.

Reply:
Here's the issue I've been referencing (if you haven't seen it): https://github.com/dart-lang/sdk/issues/33076.